csharp/samples/getting-started-receiving.md

```diff
@@ -0,0 +1,201 @@
+# Get started with Event Hubs in .NET
+
+## What will be accomplished
+
+In this tutorial, we will write a console application to receive messages from an Event Hub using the **EventProcessorHost**.
+
+## Prerequisites
```

Comment:
Do you need this though? Can't the NuGet package be consumed by the full .NET framework?